src/pages/UserHome.jsx

```diff
@@ -109,35 +107,40 @@ export default function UserHome() {
             Add Scholarship
           </Button>
         </Grid>
+        {scholarships.length === 0 && (
+          <Grid container className={classes.bannerRoot}>
+            <Grid item xs={3}>
+              <InboxIcon style={{ fontsize: 600 }} />
+            </Grid>
+            <Grid item>
+              <Typography variant="h5" gutterButtom>
+                No Scholarships Added Yet{' '}
```

Comment:
No need for this extraneous space:
```suggestion
                No Scholarships Added Yet
```